.github/workflows/server-node.yml

```
@@ -17,14 +17,14 @@ jobs:
     strategy:
       matrix:
         # Node versions to run on.
-        version: [16, 18]
+        version: [18, 24]
```

```suggestion
        version: [18, 21]
```
I put the wrong version in my suggestion. Sorry about that. :pray: